syntax = "proto3";

package second; 

service PlayMusic {
	rpc Play (MusicRequest) returns (stream MusicResponse);
}

message MusicRequest {
	string audio_name = 1; 
}

message MusicResponse {
	bytes audio_data = 1; 
}

